SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Supported Projects
+
+We will provide fixes for security issues affecting:
+
+- [Shields.io](https://shields.io)
+- Users running their own self-hosted Shields instance
+- The [badge-maker](https://www.npmjs.com/package/badge-maker) NPM package
+
+The [gh-badges](https://www.npmjs.com/package/gh-badges) NPM package is now deprecated and will no longer receive fixes for bugs or security issues.
+
+## Reporting a Vulnerability
+
+If you find a security vulnerability affecting any of our supported projects, please email [security@shields.io](mailto:security@shields.io), rather than opening a public issue on GitHub. After receiving the initial report, we will endeavor to keep you informed of the progress towards a fix and full announcement. We may ask you for additional information.
+
+Report security bugs in third-party modules to the person or team maintaining the module.
+
+## Coordinated Disclosure
+
+We aim to patch vulnerabilities within 90 days or less, disclosing the details of those vulnerabilities when a patch is published. We ask that you refrain from sharing your report with others while we work on our patch.
+
+We may want to coordinate an advisory with you to be published simultaneously with the patch, but you are also welcome to self-disclose if you prefer. We will never publish information about you or our communications with you without your permission.
```

Comment:
> but you are also welcome to self-disclose if you prefer

I'm probably being repetitive/over-explicit again, but I think it'd be worth reiterating that we'd prefer folks wait to self-disclose til after the aforementioned 90 day window.